lib/rules/no-magic-numbers.js

```diff
@@ -90,6 +95,18 @@ module.exports = {
             return ignore.indexOf(value) !== -1;
         }
 
+        /**
+         * Returns whether the number should be ignored when used as a default
+         * value assignment.
+         * @param {ASTNode} fullNumberNode `Literal` or `UnaryExpression` full number node
+         * @returns {boolean} true if the number should be ignored
```

Comment:
Same as above, it doesn't return true if the number should be ignored.

Reply:
Done.